js/songs/index.html

```diff
@@ -0,0 +1,26 @@
+<p><a data-ui-sref="welcome">Home</a></p>
+<header>
+  <form>
+    <p>Search Songs: <input type="search" label="search" id="search-box" data-ng-model="search" placeholder="search"></p>
+  </form>
+  <form data-ng-controller="NewSongController as newSongVM">
+    <p>Add a Song: <input type="text" data-ng-model="newSongVM.song.title" id="add-song-box" placeholder="add a song"><button data-ng-click="newSongVM.song.create()">Add</button>
+  </form>
+    <h4>Sort By: <button data-ng-click="songsIndexVM.sort_by('title'); songsIndexVM.is_descending = false">Title</button>
+      <button data-ng-click="songsIndexVM.sort_by('artist'); songsIndexVM.is_descending = false">Artist</button>
+      <button data-ng-click="songsIndexVM.sort_by('created_at')">Date Created </button>
+    </h4>
+
+</header>
+<div data-ng-repeat="song in songsIndexVM.songs | filter:search | orderBy: songsIndexVM.sort_on : songsIndexVM.is_descending" ng-init="songsIndexVM.sort_by('created_at')">
```

Comment:
Awesome job adding using `filter` and `data-ng-model` to search through your songs! This is one of my favorite things about Angular, just really cool :)